samples/requirements.txt

```diff
@@ -2,4 +2,5 @@
 optimum[openvino]==1.22.0
 einops==0.8.0  # For Qwen
 transformers_stream_generator==0.0.5  # For Qwen
-diffusers==0.30.3
\ No newline at end of file
+diffusers==0.30.3
+torchvision
```

Comment:
is it because of `PIL` module usage?
I suppose we need to install `pillow` then

Reply:
It was a dependency of miniCPM export script, so I temporarily added it here. It won't be needed when we switch to exporting using optimum-intel.

Reply:
ok, do we need to list `pillow` explicitly, then?

Reply:
Yes, looks like currently it is installed with torchvision, but without torchvision we will need pillow. Added it to requirements.